eq-author/src/App/QuestionnairesPage/QuestionnairesView/Header/AccessFilter.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { colors } from "constants/theme";
+import VisuallyHidden from "components/VisuallyHidden";
+
+const Fieldset = styled.div`
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  color: ${colors.textLight};
+`;
+
+const Buttons = styled.div`
+  display: flex;
+`;
+
+const FilterButton = styled.label`
+  background: white;
+  padding: 0.5rem 1rem;
+  font-size: 0.9em;
+  font-weight: bold;
+  cursor: pointer;
+  border: 1px solid ${colors.bordersLight};
+  flex: 0 0 50%;
+  text-align: center;
+  position: relative;
+  white-space: nowrap;
+
+  &:first-of-type {
+    border-radius: 4px 0 0 4px;
+    margin-right: -1px;
+  }
+
+  &:last-of-type {
+    border-radius: 0 4px 4px 0;
+  }
+
+  &:hover {
+    background: ${colors.lightMediumGrey};
+  }
+
+  &:focus-within {
+    border-color: ${colors.blue};
+    outline-color: ${colors.blue};
+    box-shadow: 0 0 0 3px ${colors.tertiary};
+  }
+
+  ${props =>
+    props.checked &&
+    css`
+      color: white;
+      background: ${colors.primary};
+      border-color: #2d5e7a;
+      z-index: 1;
+
+      &:hover {
+        background: ${colors.secondary};
+      }
+    `}
+`;
+
+const AccessFilter = ({ hideUnowned, handleToggleHideUnowned }) => {
```

Comment:
I was confused by these names and would prefer.
```suggestion
const AccessFilter = ({ filter, onFilterChanged }) => {
```
Although this would imply the filter is a constant instead of a boolean as I'm not sure it makes sense as a boolean

Reply:
I would say Sam's naming was more clear. But is it really a toggle though? Is it not just 2 buttons?

Reply:
Problem with showAll is it doesn't explain what the inverse is. I.e what does `!showAll` hide?

Reply:
Ah thats true. Just stick with what you think will be best then.